Makefile

```diff
@@ -279,10 +279,13 @@ uninstall:
 	- kubectl delete clusterroles.rbac.authorization.k8s.io "system:controller:operator-lifecycle-manager"
 	- kubectl delete clusterrolebindings.rbac.authorization.k8s.io "olm-operator-binding-openshift-operator-lifecycle-manager"
 
-.PHONY: run-local
-run-local: build-linux build-wait build-util-linux
+.PHONY: build-local
+build-local: build-linux build-wait build-util-linux
```

Comment:
Note: I don't think this phony target is needed but I added it anyways to maintain consistency with the rest of the Makefile targets.